sites/fast-website/src/app/components/feature-card/feature-card.styles.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from "@microsoft/fast-element";
+import { display, neutralForegroundRestBehavior } from "@microsoft/fast-components";
+
+export const FeatureCardStyles = css`
+    ${display("grid")} :host {
+        contain: layout;
+        grid-template-columns: 1fr 2fr;
+        color: var(--neutral-foreground-rest);
+        font-family: var(--text-font);
```

Comment:
where does this var exist? We may want to simply inherit unless there are alternate fonts being loaded and used.

Reply:
I saw it being used in the main css file and assumed it was going to be established in :root.  Inherit works just as well.  I will change it to inherit.